src/shadowbox/model/access_key.ts

```diff
@@ -53,6 +53,8 @@ export interface AccessKeyRepository {
   createNewAccessKey(): Promise<AccessKey>;
   // Removes the access key given its id. Throws on failure.
   removeAccessKey(id: AccessKeyId);
+  // Returns the access key with the given id. Throws on failure.
+  getAccessKey(id: AccessKeyId): AccessKey;
```

Comment:
We need to update https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/server/api.yml too

Reply:
@fortuna

FYI https://github.com/Jigsaw-Code/outline-server/pull/1142/commits/234155e9ae6438c39e25cc42f874a3dd0af800f0